src/wallet/scriptpubkeyman.cpp

```diff
@@ -1966,6 +1966,10 @@ bool DescriptorScriptPubKeyMan::SetupDescriptorGeneration(const CExtKey& master_
         desc_prefix = "tr(" + xpub  + "/86'";
         break;
     }
+    case OutputType::UNKNOWN: {
+        // Do nothing if the address is of UNKNOWN OutputType
+        return false;
+    }
```

Comment:
Could assert(false) here.
`DescriptorScriptPubKeyMan::SetupDescriptorGeneration` is called from `CWallet::SetupDescriptorScriptPubKeyMans` and this latter function is already skipping this type.

Reply:
good point, `SetupDescriptorGeneration` is actually defined on the `DescriptorScriptPubKeyMan` and we should never have a spk_m setup for the unknown type, so assert(false) is better than just returning a bool.